packages/@aws-cdk-testing/framework-integ/test/aws-ec2/test/integ.instance-init.js.snapshot/EnableSumTimeoutInstanceTestDefaultTestDeployAssert5CD5B9B5.template.json

```diff
@@ -0,0 +1,36 @@
+{
+ "Parameters": {
+  "BootstrapVersion": {
+   "Type": "AWS::SSM::Parameter::Value<String>",
+   "Default": "/cdk-bootstrap/hnb659fds/version",
+   "Description": "Version of the CDK Bootstrap resources in this environment, automatically retrieved from SSM Parameter Store. [cdk:skip]"
+  }
+ },
+ "Rules": {
+  "CheckBootstrapVersion": {
+   "Assertions": [
+    {
+     "Assert": {
+      "Fn::Not": [
+       {
+        "Fn::Contains": [
+         [
+          "1",
+          "2",
+          "3",
+          "4",
+          "5"
+         ],
+         {
+          "Ref": "BootstrapVersion"
+         }
+        ]
+       }
+      ]
+     },
+     "AssertDescription": "CDK bootstrap stack version 6 required. Please run 'cdk bootstrap' with a recent version of the CDK CLI."
+    }
+   ]
+  }
+ }
+}
```

Comment:
What is this assert stack/template? I don't see any assert used. Can you remove the snapshot folder and re-run the tests again?

Reply:
Correct me if I'm wrong but I believe that these are generated because of using `integ.IntegTest`, passing in the stack as the test case as follows:
```
new integ.IntegTest(app, 'instance-init-test', {
  testCases: [stack],
});
```
Since I deleted the snapshot folder and regenerated, it registered as a new test and therefore needed `IntegTest` construct to configure the test